pandas/core/ops/blockwise.py

```diff
@@ -0,0 +1,92 @@
+from typing import TYPE_CHECKING, List, Tuple
+
+import numpy as np
+
+from pandas._typing import ArrayLike
+
+if TYPE_CHECKING:
+    from pandas.core.internals.blocks import Block  # noqa:F401
+
+
+def operate_blockwise(left, right, array_op):
+    assert right._indexed_same(left)
```

Comment:
This is already asserted where this `operate_blockwise` is called as well, so one of both should be sufficient?

Reply:
good idea